app/src/test/java/br/org/cesar/discordtime/stickysessions/navigation/RouteRepositoryTest.kt

```diff
@@ -0,0 +1,55 @@
+package br.org.cesar.discordtime.stickysessions.navigation
+
+import br.org.cesar.discordtime.stickysessions.navigation.repository.IRouteRepository
+import br.org.cesar.discordtime.stickysessions.navigation.repository.RouteRepository
+import br.org.cesar.discordtime.stickysessions.navigation.router.IRouter
+import br.org.cesar.discordtime.stickysessions.navigation.router.Route
+import br.org.cesar.discordtime.stickysessions.ui.lobby.LobbyActivity
+import br.org.cesar.discordtime.stickysessions.ui.session.SessionActivity
+import com.nhaarman.mockito_kotlin.atLeastOnce
+import com.nhaarman.mockito_kotlin.verify
+import com.nhaarman.mockito_kotlin.whenever
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+
+class RouteRepositoryTest {
+
+    private lateinit var routeRepository: IRouteRepository
+
+    @Before
+    fun setUp(){
+        routeRepository = RouteRepository()
+    }
+
+    @Test(expected = IllegalArgumentException::class)
+    fun `should throw IllegalArgument if from is null `() {
+        var route = routeRepository.findRoute(null, "")
+        Assert.assertNull(route)
+    }
+
+    @Test(expected = IllegalArgumentException::class)
+    fun `should throw IllegalArgument if event is null `() {
+        var route = routeRepository.findRoute("", null)
+        Assert.assertNull(route)
+    }
+
+    @Test
+    fun `should return null if route is invalid `() {
+        var route = routeRepository.findRoute("d", "d")
+        Assert.assertNull(route)
+    }
+
+    @Test
+    fun `should return SessionActivity class if from is LobbyActivity and event is createdSession `() {
```

Comment:
To test if every route is being returned correctly I would instead create a parameterized test class. Please refer to: https://github.com/junit-team/junit4/wiki/parameterized-tests